dm/master/server_test.go

```diff
@@ -1655,6 +1616,11 @@ func (t *testMaster) TestJoinMember(c *check.C) {
 }
 
 func (t *testMaster) TestOperateSource(c *check.C) {
```

Comment:
testMaster is not a `SerialSuites`, maybe we should access `checkAndAdjustSourceConfigFunc` in `SetUpSuite` to avoid race

Reply:
fixed in https://github.com/pingcap/dm/pull/2071/commits/86ebb8787bbe2bcbc0bcae7aa38d5a3b2b072749